src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -181,23 +185,48 @@ private EpisodeDto MapEpisodeToEpisodeDto(BssEpisode episode)
 
     private async Task ProcessParticipantDataAsync(IEnumerable<Participant> participants, string participantUrl)
     {
+
+        DateTime processingStart = DateTime.UtcNow;
+
+        int successCount = 0;
+        int failureCount = 0;
+        int lastProcessedRow = 0;
+        int rowIndex = 0;
+
         try
         {
-            _logger.LogInformation("Processing participant data.");
+            _logger.LogInformation("Processing started for file at {processingStart}", processingStart);
             foreach (var participant in participants)
             {
                 string serializedParticipant = JsonSerializer.Serialize(participant, new JsonSerializerOptions { WriteIndented = true });
 
                 _logger.LogInformation("Sending participant to {Url}: {Request}", participantUrl, serializedParticipant);
 
                 await _httpRequestService.SendPost(participantUrl, serializedParticipant);
+
+                successCount++;
+                rowIndex++;
+                lastProcessedRow = rowIndex;
             }
+
+
         }
         catch (Exception ex)
         {
             _logger.LogError(ex, "Error in ProcessParticipantDataAsync: {Message}", ex.Message);
+            failureCount++;
             await ProcessParticipantDataAsync(participants, participantUrl);
         }
+
+        DateTime processingEnd = DateTime.UtcNow;
+        if (failureCount == 0)
+        {
+            _logger.LogInformation( "Rows Processed: {successCount}:",successCount);
+        }
+        else
+        {
+            _logger.LogWarning("Last Successful Row: {lastProcessedRow}, Rows Processed: {successCount}, Failures: {failureCount}", lastProcessedRow,successCount, failureCount);
+        }
```

Comment:
Regardless of the failureCount, we should log both successCount and failureCount. We should also be logging the file name.

Reply:
fileName, successCount, failureCount will be logged.

Reply:
I think we can get rid of the if else statement and log the same message either way. The only current difference between the two is that one includes the lastProcessedRow but it's not necessary to log that here since it's already logged above after each row is processed.